File: Java codes/OOP codes/first program/src/Calculator.java
import java.util.Scanner;
public class Calculator {
    public static double addition(double num1, double num2){
        return num1+num2;
    }

    public static double subtraction(double num1, double num2){
        return num1-num2;
    }

    public static double multiplication(double num1, double num2){
        return num1*num2;
    }

    public static double division(double num1, double num2){
        return num1/num2;
    }

    public static double remainder(double num1, double num2){
        return num1%num2;
    }

    public static void main(String[] args) {
        Scanner sc= new Scanner(System.in);

        double num1, num2;
        byte op;

        System.out.print("Enter First Number = ");
        num1 = sc.nextDouble();

        System.out.print("Enter Second Number = ");
        num2 = sc.nextDouble();

        do {

        System.out.println("Press 1 for '+' .\nPress 2 for '-' .");
        System.out.println("Press 3 for '*' .\nPress 4 for '/' .\nPress 4 for '/' .");
        op = sc.nextByte();

        switch (op){
            case 1:
                System.out.println("SUM = " + Calculator.addition(num1,num2));
                break;

            case 2:
                System.out.println("Subtraction = " + Calculator.subtraction(num1,num2));
                break;

            case 3:
                System.out.println("Multiplication = " + Calculator.multiplication(num1,num2));
                break;

            case 4:
                System.out.println("Division = " + Calculator.division(num1,num2));
                break;

            case 5:
                System.out.println("Remainder = " + Calculator.remainder(num1,num2));
                break;

            default:
                System.out.println("ERROR... Invalid input");
                break;
        }
    }while (op<0 || op>6);
}}

File: C++ codes/calculator.cpp
#include<stdio.h> 
#include <math.h>
#include <windows.h>


int main () 
{
		system("color 2f");
		
    printf("\t SCIENTIFIC CALCULATOR\n\n\n");
    char choice; 
    
    printf("Enter the number which mathematical operation do you want to perform\n");
    printf("For Addition press '1'\nFor Subtraction press '2'\nFor Multiplication press '3'\nFor Division press '4'\n");
    printf("For Finding Power press '5'\nFor Finding SIN press '6'\nFor Finding COS press '7'\nFor Finding TANGENT press '8'\nFor Finding UNDER ROOT press '9'\n");
	scanf("%c",&choice);
    
    system ("CLS");
    float val1, val2, result;
    
    switch (choice)
    {
    	case '1':
    		
    		printf("Enter the first value (addition): "); 
            scanf("%f", &val1);
            
           	printf("\nEnter the second value (addition): "); 
            scanf("%f", &val2);
            
    		result = val1+val2;
    		printf("\nAddition of %f and %f = %f",val1, val2, result);
    		break;
    		
    	case '2':
    		printf("\nEnter the first value (subtraction): "); 
            scanf("%f", &val1); 
            
            printf("\nEnter the second value (subtraction): "); 
            scanf("%f", &val2); 
            
			result = val1-val2;
    		printf("\nSubtraction of %f and %f = %f",val1, val2, result);
    		break;
    		
    	case '3':
    		printf("\nEnter the first value (multiplication): "); 
            scanf("%f", &val1); 
            
			printf("\nEnter the second value (multiplication): "); 
            scanf("%f", &val2); 
            
			result=val1*val2;
    		printf("\nMultiplication of %f and %f = %f",val1, val2, result);
    		break;
			
		case '4':
			printf("\nEnter the first value (division): "); 
            scanf("%f", &val1); 
            
			printf("\nEnter the second value (division): "); 
            scanf("%f", &val2); 
            
			result=val1/val2; 
            printf("\nDivision of %f and %f = %f",val1, val2, result);
    		break;
			
		case '5':
			printf("Enter the base value: "); 
            scanf("%f", &val1); 
            
			printf("\nEnter the exponent value: "); 
            scanf("%f", &val2); 
			 
            result=pow(val1,val2);
            printf("\nThe power of %f numbers is: %f ", val1,pow(val1,val2));
    		break;
			
		case '6':
			printf("Enter the value to find SIN: "); 
            scanf("%f", &val1); 

            result=sin(val1);
            printf("\nThe sin of %f numbers is :%f",val1, result);
    		break;
			
		case '7':
			printf("Enter the value to find COS: "); 
            scanf("%f", &val1);  
            
			result=cos(val1); 
            printf("\nThe COS of %f numbers is :%f",val1, result);
    		break;
			
		case '8':
			printf("Enter the value to find TANGENT: "); 
            scanf("%f", &val1); 

            result=tan(val1); 
            printf("\nThe tangent of %f numbers is :%f",val1, result);
    		break;
			
		case '9':
			printf("Enter the value to find UNDER ROOT: "); 
            scanf("%f", &val1); 

            result=sqrt(val1); 
            printf("\nThe Under_root of %f numbers is :%f",val1, result);
    		break;	
			
		default:
			printf("INVALID INPUT\nTRY AGAIN... ");	
			break;				
	}

}
